packages/core/src/actors/callback.ts

```diff
@@ -50,10 +50,12 @@ export function fromCallback<TEvent extends EventObject>(
           state.receivers.add(newListener);
         };
 
-        state.dispose = invokeCallback(sender, receiver, {
+        state.dispose = invokeCallback({
           input: state.input,
           system,
-          self: self as TODO
+          self: self as TODO,
+          sendBack: sender,
+          receive: receiver
```

Comment:
It would make sense to rename the local variables to use the same names as the ones that we expose to the user now